set.mm

```diff
@@ -837895,6 +838014,1624 @@ have GLB (expanded version).  (Contributed by Zhi Wang,
     $}
   $}
 
+  $c UP $.
+
+  $( Extend class notation with the class of universal properties. $)
+  cup $a class UP $.
+
+  ${
+    $d B b c d e f g h j k m n o t u w x y z $.
+    $d C b c d e f g h j k m n o t u w x y z $.
+    $d D b c d e f g h j k m n o t u w x y z $.
+    $d E b c d e f g h j k m n o t u w x y z $.
+    $d F b c d e f g h j k m n o t u w x y z $.
+    $d G b c d e f g h j k m n o t u w x y z $.
+    $d H b c d e f g h j k m n o t u w x y z $.
+    $d J b c d e f g h j k m n o t u w x y z $.
+    $d M b c d e f g h j k m n o t u w x y z $.
+    $d N b c d e f g h j k m n o t u w x y z $.
+    $d O b c d e f g h j k m n o t u w x y z $.
+    $d W b c d e f g h j k m n o t u w x y z $.
+    $d X b c d e f g h j k m n o t u w x y z $.
+    $d Y b c d e f g h j k m n o t u w x y z $.  $d m ph x $.
+    $( Definition of the class of universal properties.  (Contributed by Zhi
```

Comment:
I think some more informatiuon would be helpful here: universal properties for what? See also the following comments.